src/App.tsx

```diff
@@ -31,48 +18,16 @@ const App: FunctionComponent = () => {
             <main>
                 <Switch>
                     <Route path="/stillinger">
-                        <Microfrontend<FellesMicrofrontendProps>
-                            key="rekrutteringsbistand-stilling"
-                            appName="rekrutteringsbistand-stilling"
-                            appPath="/rekrutteringsbistand-stilling"
-                            appProps={{
-                                navKontor,
-                                history,
-                            }}
-                        />
+                        <Stilling navKontor={navKontor} history={history} />
                     </Route>
                     <Route path="/stillingssok">
-                        <Microfrontend<FellesMicrofrontendProps>
-                            key="rekrutteringsbistand-stillingssok"
-                            appName="rekrutteringsbistand-stillingssok"
-                            appPath="/rekrutteringsbistand-stillingssok"
-                            appProps={{
-                                navKontor,
-                                history,
-                            }}
-                        />
+                        <Stillingssøk navKontor={navKontor} history={history} />
                     </Route>
                     <Route path="/kandidater">
-                        <Microfrontend<FellesMicrofrontendProps>
-                            key="rekrutteringsbistand-kandidat"
-                            appName="rekrutteringsbistand-kandidat"
-                            appPath="/rekrutteringsbistand-kandidat"
-                            appProps={{
-                                navKontor,
-                                history,
-                            }}
-                        />
+                        <Kandidat navKontor={navKontor} history={history} />
                     </Route>
-                    <Route exact path="/">
-                        <Microfrontend<FellesMicrofrontendProps>
-                            key="rekrutteringsbistand-statistikk"
-                            appName="rekrutteringsbistand-statistikk"
-                            appPath="/rekrutteringsbistand-statistikk"
-                            appProps={{
-                                navKontor,
-                                history,
-                            }}
-                        />
+                    <Route path="/">
```

Comment:
Er det noen grunn for at vi ikke har statistikksiden med `exact` lengre?

Reply:
Øøh nei! Det var ikke meningen, slenger den på igjen